imagersite/photo/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from .views import album_view, photo_view, library_view, add_view, edit_view, faces_view
```

Comment:
So wait a minute.  If you have these views here, why are they also over in the profile app?